### DISCUSSION TITLE
我在使用tweakeroo模组时遇到一个问题

### DISCUSSION THREAD

**Discussion by Pangzongawa** [maruohon/tweakeroo#519] [Q&A]
我在使用一个生电整合包时遇到一个问题，详见下方视频

https://github.com/user-attachments/assets/00171a34-3568-42bb-93af-477bdb7d7141

就是放置的一些方块不会被显示

我排查了其他模组，在安装有malilib+tweakeroo时会出现问题，在安装malilib+tweakmore时候并没出现问题

接着我看了看tweakeroo的设置，应该没有导致该问题出现的设置

希望能有大佬来帮下忙，谢谢了

**Comment by wendavid552** [comment]
具体来讲，是只安装了malilib和tweakeroo吗，还有什么其他的模组呢？最好能提供一个最小复现模组列表，也就是最少只使用几个模组的时候会出现这种情况？或者如果你不想尝试的话也至少发一下模组列表出来。

**Reply by Pangzongawa** [reply]
后面测了只加了malilib和tweakeroo的，会复现此问题
